.github/workflows/_e2e.yml

```diff
@@ -79,7 +79,7 @@ jobs:
 
       - name: Build apps-backend
         if: inputs.isAppsBackend || github.ref_name == 'develop'
-        run: pnpm --filter apps-backend build
+        run: pnpm turbo --filter apps-backend build
```

Comment:
Mind explaining this to someone who doesn't use this stuff much?

Reply:
sure! this is for turborepo to build the SDK so that it can be accessible in apps-backend :pray: https://github.com/iotaledger/iota/pull/4764#pullrequestreview-2549233325

Reply:
Some more details on the difference.
**Without** `turbo` it uses pnpm to run the `build` task in the `apps-backend` package. 
**With** `turbo` it goes through turbo, so it will respect the task hierarchy defined in `turbo.json` and run then dependent tasks first if needed (cache miss).